easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -165,9 +203,9 @@ def configure_step(self):
 
     def build_step(self):
         """Build Python package using setup.py"""
-
-        cmd = "%s python setup.py build %s" % (self.cfg['prebuildopts'], self.cfg['buildopts'])
-        run_cmd(cmd, log_all=True, simple=True)
+        if not self.cfg['use_easy_install']:
```

Comment:
@ocaisa: this may be the culprit... If we use the below, you shouldn't run into the problem you're having.

```python
if not self.cfg.get('use_easy_install', None):
```

The problem is that `use_easy_install` is not a general easyconfig parameter, but one specific to `PythonPackage`.

Although, even with that in mind, this line should never yield the error you're seeing...

Can you provide a full (debug) log?